notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -7,8 +7,8 @@
     "id": "rQc-wXjqrEuR"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    def transform_fn(data_item):
</pre><p>I think we need to add info about nncf.Dataset and transform_fn.</p><p>Also I think it is necessary explain why model_type parameter used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/941/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>